core/src/utilities/helpers/generic-helpers.js

```diff
@@ -252,7 +252,14 @@ class GenericHelpersClass {
   }
 
   getContentAreaHeight /* istanbul ignore next */() {
-    return this.getInnerHeight() - LuigiElements.getShellbar().clientHeight;
+    const contentAreaHeight = this.getInnerHeight() - this.getShellbarHeight();
+    return contentAreaHeight;
+  }
+
+  getShellbarHeight() {
```

Comment:
Can you add some JSDoc comment to the new function you added. Check the other examples in this file, similar to the following: 

```
  /**
   * {Function description}
   *
   * @returns {return description}
   */
```

Reply:
Added some JSDoc comments for the new function, `getShellbarHeight`. See [aaed82f](https://github.com/SAP/luigi/pull/3301/commits/aaed82f835ff90dda1e439571438c2d480c25ace).